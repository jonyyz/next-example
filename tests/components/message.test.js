import { render } from "@testing-library/react";
import Message from "@components/message.js";

describe("Message Component", () => {
  it("should render", () => {
    const { container } = render(<Message message="test" />);
    expect(container).toMatchSnapshot();
  });
});
