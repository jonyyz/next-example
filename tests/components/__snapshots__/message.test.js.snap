// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message Component should render 1`] = `
<div>
  <div>
    test
  </div>
</div>
`;
